builder/builder.go

```diff
@@ -609,3 +609,104 @@ func (b *Builder) BuildDeltaPacksPreviousVersions(prev, to uint32, printReport b
 	}
 	return nil
 }
+
+// BuildDeltaManifests between two versions of the mix.
+func (b *Builder) BuildDeltaManifests(from, to uint32) error {
+	var err error
+
+	if to == 0 {
+		to = b.MixVerUint32
+	} else {
+		if to > b.MixVerUint32 {
```

Comment:
Nit: Could be elise if

Reply:
Ah indeed, copypasta'd logic and didn't think too much about the extra if, fixing.